src/software/thunderscope/common/common_widgets.py

```diff
@@ -233,8 +319,28 @@ def enable_slider(slider, label, get_value):
 
 
 def disable_radio_button(button_group):
+    """
+    Disables a whole radio button group
+    Sets all buttons to unselected and disables their onClick function
+    :param button_group: button group to disable
+    """
     button_group.setExclusive(False)
     for button in button_group.buttons():
         button.setChecked(False)
         button.clicked.disconnect()
         button.clicked.connect(lambda state, curr=button: curr.setChecked(False))
+
+
+def draw_robot(painter, rect, start_angle, span_angle):
+    """
+    Draws a robot bounded by the given rectangle with a chord defined by the given angles
+    :param painter: the QPainter object that is drawing in thunderscope
+    :param rect: the rectangle that is bounding the robot's circle
+    :param start_angle: the start of the chord, measured anti-clockwise from the horizontal middle
+    :param span_angle: the end of the chord, measured anti-clockwise from the horizontal middle
+    """
+    convert_degree = -16
+
+    painter.drawChord(
+        rect, start_angle * convert_degree, span_angle * convert_degree,
+    )
```

Comment:
refactor start_angle, span_angle so that it includes the units in the param itself (eg: span_angle_rad, span_angle_degrees, etc)

Reply:
done